### ISSUE TITLE
Add a payroll system, just as an indicator, not as direct payment system

### ISSUE THREAD

**Issue by SimonSarazin** [liberapay/liberapay.com#64]
We could provide a tool to help people to distribute the money in a team as Gratipay invented it, where everyone sets their own compensation 
https://gratipay.com/about/features/payroll

This tool could look at what Gratipay proposed. That's what i actually do on my projects, but using spreasheets. I now have 6 spreadsheet, one for each project... It could be super to get it directly on liberapay : https://docs.google.com/spreadsheets/d/153wK4wyuMrqplYIakrCWh34dDRzik_56_F6UxjnW-R4/edit?usp=sharing

The owner could after that decide how to give this money, for example by asking an invoice. This tool could only be an indicator, not a payment system, so each team manage his own way to pay people, with his own legal structure.

**Comment by Changaco** [comment]
I think I understand, but first I need to clarify something. Liberapay has 3 types of accounts: individual, organization, and teams. Teams are only for informal organizations, they have no owner, all participants are equal. This issue is not about team accounts.

What Simon seems to be asking here is for organization accounts to work almost like the old Gratipay teams, except without money actually changing hands. That means adding new features to organization accounts:

- the ability for the organization to have members
- the ability for members to ask the team for money
- the ability for both to see and modify the status of each request (cancel, reject, accept, mark as paid)

Am I understanding you correctly @SimonSarazin?

**Comment by SimonSarazin** [comment]
Hi Changaco, i think it's exactly that :-) That's our actual usage.

It's maybe an other thread, but i don't clearly see what will be the difference between Gratipay and Liberapay. Is there any information about it ? In order not to ask to the two plateform to develop this feature :-)
Maybe to discuss in an other thread, @whit537 and @Changaco, but we'd like to invit you in Lille when @whit537 will come in France, as we organise an event on the "take what you want" idea, in preparation of an event in the north of France that we organise since 10 years (http://roumics.com).

**Comment by whit537** [comment]
> we'd like to invit you in Lille 

Thanks for the invite, @SimonSarazin! Maybe we could coordinate on https://github.com/gratipay/inside.gratipay.com/issues/464?

**Comment by SimonSarazin** [comment]
I'm giving more information about this issue.

I speak in this issue about "teams".

The idea it to be able to use the "pay what you want" system in a team, but separated from any payment system. Exactly as it works with cobudget, where it's just a tool, not a payment tool. 
I think this could be used by lots of teams who just want to start the "pay what you want" inside their team, not the whole liberapay service. This could be a first step. For example, in ANIS, we can't for the moment directly put a lot of money on liberapay for juridical questions that we have to secure (that's why we test with a little amount), but we want to use the "take what you want" tool to distribute the money inside our community (and only after that we will ask people to make a bill). 



Maybe just by adding a new currency that any team can create inside his own team to distribute ?

**Comment by Changaco** [comment]
@SimonSarazin just asked about this in IRC.

> Maybe just by adding a new currency

We don't support multiple currencies, not even virtual ones, so we can't "just add a new currency".

> The idea it to be able to use the "pay what you want" system in a team, but separated from any payment system. Exactly as it works with cobudget, where it's just a tool, not a payment tool.

I don't see this being implemented in either Liberapay or Gratipay anytime soon, we're both payment platforms, not abstract tools like cobudget, so I think it would have to be a separate service (either a new one or an existing one like cobudget with a take-what-you-want system added).

**Comment by Changaco** [comment]
Closing in favor of #505.
